smac/epm/gaussian_process_mcmc.py

```diff
@@ -236,15 +244,22 @@ def _train(self, X: np.ndarray, y: np.ndarray, do_optimize: bool=True):
             kernel = deepcopy(self.kernel)
             kernel.theta = self.p0
             model = GaussianProcess(
+                configspace=self.configspace,
                 types=self.types,
                 bounds=self.bounds,
                 kernel=kernel,
-                normalize_y=self.normalize_y,
+                normalize_y=False,
                 seed=self.rng.randint(low=0, high=10000),
             )
             model._train(X, y, do_optimize=False)
             self.models.append(model)
 
+        if self.normalize_y:
+            for model in self.models:
```

Comment:
Don't get this. First you are normalizing outside of model, now the mode itself should normalize. Why?

Reply:
Also is this now the same as for the other GP class?

Reply:
> First you are normalizing outside of model
This makes sure that the data is exactly the same when passed to scikit-learn to compute the likelihoods, and when passed to the SMAC GP class.
> now the mode itself should normalize
No, the model is already trained at this point in time. This basically makes sure that the base models denormalize the data themselves, and that this is not done by the GPMCMC class.

If that makes sense I'll add a few more comments to avoid such confusion in the future.